models/userDao.js

```diff
@@ -1 +1,38 @@
-const dataSource = require('./dataSource')
\ No newline at end of file
+const dataSource = require('./dataSource')
+
+const getUserBySocialId = async (socialId) => {
+
+	const [user] = await dataSource.query(`
+		SELECT 
+            id,
+            social_id,
+            name,
+            birth,
+            profile_image
+		FROM users
+		WHERE social_id=?`, [socialId]
+	)
```

Comment:
- SQL 문법도 가독성 좋게 만들어주는 것이 중요합니다. 회사마다 컨벤션이 다르겠지만, 위코드에서는 아래와 같은 SQL 문법의 컨벤션을 정하고 있습니다.
- 지금은 위코드에서 사용하고 있는 컨벤션을 지켜서 문법을 작성해주세요. 하지만 이는 회사마다 다르기 때문에, 정답이 아님을 반드시 인지해주세요.
- SQL 포맷팅은 다음 처럼 통일하면 좋을거 같습니다.

```suggestion
	const [user] = await dataSource.query(`
	    SELECT 
	        id,
	        social_id,
	        name,
	        birth,
	        profile_image
	    FROM users
	    WHERE social_id=?
	    `, [socialId]
	)
```

Reply:
vscode상에서는 후자처럼 작성되어  있는데 github상에서 들여쓰기가 이상하게 나옵니다..